corehq/apps/app_manager/templates/app_manager/v2/form_view.html

```diff
@@ -8,7 +8,7 @@
 {% endblock %}
 
 {% block case_management_js %}{{ block.super }}
-  {% if form.uses_cases %}
+  {% if form.source %}
```

Comment:
@orangejenny I changed this because the surrounding logic that required it used the same requirements. I guess I didn't take usercase into account